### PR TITLE
Fixes #2511 - Triage dashboard 'needsinfo' label category doesn't include 'needsinfo-xxx' labels

### LINKED ISSUE #2511: Triage Dashboard Needsinfo card doesn't take into account Needsinfo-xxx labels
**Environment:**
Browser / Version: Chrome 67 (Desktop), Firefox Nightly 62.0a1 (Desktop)
Operating System: Windows 10

**Steps to Reproduce:**
1. Go to https://github.com/webcompat/web-bugs/milestone/2
2. Ctrl+F (or Command+F) needsinfo. Observe the total number of labels that contain this string.
3. Go to https://webcompat.com/dashboard/triage
4. Observe the information displayed in the Needsinfo card.

**Expected Behavior:**
To be decided.

**Actual Behavior:**
The information is not consistent. The Needsinfo-xxx labels are not taken into account.

**Notes:**
1. Screen capture attached.
2. Reproducible both on Chrome and Firefox.

Watchers:
@softvision-sergiulogigan
@softvision-oana-arbuzov

sv;

image

### PR DESCRIPTION
**Pull request by laghee**
This PR fixes issue `#2511`

## Proposed PR background

<p>We're now filtering on anything that starts with `status-needsinfo`, so this should  cover future pingees as well.</p>
---

- [✅] I have read the [Pull Request + Code Style Guidelines](https://github.com/webcompat/webcompat.com/blob/master/docs/pr-coding-guidelines.md) thoroughly.

### CODE REVIEW

File: webcompat/dashboard.py [hunk 1]
```diff
@@ -42,17 +42,19 @@ def filter_needstriage(milestone_list):
...
     return needstriage_list, dashboard_stats
 
 
 def has_needsinfo(labels):
     """Assess if the issue has a needsinfo label."""
-    needsinfo = False
-    if 'status-needsinfo' in labels:
-        needsinfo = True
-    return needsinfo
+    print(labels)
```
Comment (karlcow) [inline webcompat/dashboard.py]: nit: oops ;)

File: webcompat/dashboard.py [hunk 2]
```diff
@@ -42,17 +42,19 @@ def filter_needstriage(milestone_list):
...
+    needsinfo = (label for label in labels if label.startswith(
+        'status-needsinfo'))
```
Comment (karlcow) [inline webcompat/dashboard.py]: nit: not very important, but this is probably more readable.

```python
needsinfo = (label for label in labels 
             if label.startswith('status-needsinfo'))
```

File: webcompat/dashboard.py (hunk 1 above)
Comment (laghee) [reply webcompat/dashboard.py]: lol ... oops 🤦‍♀️

File: webcompat/dashboard.py (hunk 2 above)
Comment (laghee) [reply webcompat/dashboard.py]: I agree

**Pull request merged**